package.json

```diff
@@ -26,6 +26,6 @@
   ],
   "devDependencies": {
     "eslint-config-enact-proxy": "^1.0.7",
-    "lerna": "^7.4.2"
+    "lerna": "^8.1.2"
   }
-}
\ No newline at end of file
+}
```

Comment:
please add a block for node restrictions, vased on the breaking changes

![image](https://github.com/enactjs/templates/assets/63335068/0b7823d0-5435-487f-9692-e7b2b97c6265)